From dafdd2b2b0579ada83a53b1a20cfa1bc552ef7e6 Mon Sep 17 00:00:00 2001
Message-ID: <dafdd2b2b0579ada83a53b1a20cfa1bc552ef7e6.1705743627.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 23:03:16 +0000
Subject: [PATCH 09/38] gnu: go-github-com-marten-seemann-chacha20: Move to
 (gnu packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-marten-seemann-chacha20): Move
from here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: I25bc41e79e119f7a91510d5b7c55466ad7073a5c
---
 gnu/packages/golang-crypto.scm | 23 +++++++++++++++++++++++
 gnu/packages/golang.scm        | 22 ----------------------
 2 files changed, 23 insertions(+), 22 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 4462eb6141..93123b547b 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -1,5 +1,6 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2018 Pierre Neidhardt <mail@ambrevar.xyz>
+;;; Copyright © 2020 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;;
 ;;; This file is part of GNU Guix.
@@ -128,6 +129,28 @@ (define-public go-github-com-libp2p-go-libp2p-peer
       (description "PKI based identities for use in @command{go-libp2p}.")
       (license license:expat))))
 
+(define-public go-github-com-marten-seemann-chacha20
+  (package
+    (name "go-github-com-marten-seemann-chacha20")
+    (version "0.2.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/marten-seemann/chacha20")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0x1j4cvbap45zk962qkjalc1h3axhzzdy9cdzhcjmprmm1ql4gjm"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/marten-seemann/chacha20"))
+    (home-page "https://github.com/marten-seemann/chacha20")
+    (synopsis "ChaCha20 in Go")
+    (description "This package is an external copy of the Go standard
+library's internal ChaCha20 package.")
+    (license license:bsd-3)))
+
 (define-public go-github-com-multiformats-go-multihash
   (let ((commit "97cdb562a04c6ef66d8ed40cd62f8fbcddd396d6")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 358edfed63..ab6bba3697 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -7126,28 +7126,6 @@ (define-public go-github-com-marten-seemann-qtls
     (home-page "https://github.com/marten-seemann/qtls")
     (license license:bsd-3)))
 
-(define-public go-github-com-marten-seemann-chacha20
-  (package
-    (name "go-github-com-marten-seemann-chacha20")
-    (version "0.2.0")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                     (url "https://github.com/marten-seemann/chacha20")
-                     (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "0x1j4cvbap45zk962qkjalc1h3axhzzdy9cdzhcjmprmm1ql4gjm"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/marten-seemann/chacha20"))
-    (synopsis "ChaCha20 in Go")
-    (description "This package is an external copy of the Go standard library's
-internal ChaCha20 package.")
-    (home-page "https://github.com/marten-seemann/chacha20")
-    (license license:bsd-3)))
-
 (define-public go-github-com-cheekybits-genny
   (package
     (name "go-github-com-cheekybits-genny")
-- 
2.41.0

